lib/resque_web/version.rb

```diff
@@ -1,3 +1,3 @@
 module ResqueWeb
-  VERSION = "0.0.12"
+  VERSION = "0.0.13"
```

Comment:
Revert version, this can be done separately

```suggestion
  VERSION = "0.0.12"
```